### PR TITLE
Migrated test_healed_and_heal_failed_command

### PR DESCRIPTION
**Pull request by aujjwal-redhat**
Migrated
[test_healed_and_heal_failed_command](https://github.com/gluster/glusto-tests/blob/HEAD/tests/functional/afr/test_healed_and_heal_failed_command.py)

Updates: #292

Signed-off-by: Ayush Ujjwal <aujjwal@redhat.com>

### CODE REVIEW

**Comment by aujjwal-redhat** [comment]
```sh

Results for afr
Test nature : disruptive
TestName : test_healed_and_heal_failed_command
+-------------+-------------+-----------------------+-------------+
| Volume Type | Test Result | Time Taken (hh:mm:ss) | Skip Reason |
+-------------+-------------+-----------------------+-------------+
|     rep     |     PASS    |        00:02:18       |      NA     |
+-------------+-------------+-----------------------+-------------+
```

File: tests/functional/afr/test_healed_and_heal_failed_command.py [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+
+        # Verify `gluster volume heal <volname> info healed` results in error
+        cmd = 'gluster volume heal <volname> info'
+        ret = redant.heal_info_healed(self.vol_name, self.server_list[0],
+                                      False)
+        if ret['error_code'] == 0:
+            raise Exception(f"{cmd} heal-failed should result in error.")
+
+        if 'Usage' not in ret['error_msg']:
+            raise Exception(f"{cmd} heal-failed should list 'Usage'")
```
Comment (nik-redhat) [inline tests/functional/afr/test_healed_and_heal_failed_command.py:89]: You have missed the `healed_failed` part but added the check for them. Please rectify.

**Review by nik-redhat — CHANGES_REQUESTED** [review]

File: tests/functional/afr/test_healed_and_heal_failed_command.py (hunk 1 above)
Comment (aujjwal-redhat) [reply tests/functional/afr/test_healed_and_heal_failed_command.py:89]: Ack.

File: tests/functional/afr/test_healed_and_heal_failed_command.py [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+
+        except Exception as error:
+            tb = traceback.format_exc()
+            self.redant.logger.error(error)
+            self.redant.logger.error(tb)
+        super().terminate()
+
+    def run_test(self, redant):
+        """
+
```
[the remark this answers is not in the document]
Comment (aujjwal-redhat) [reply tests/functional/afr/test_healed_and_heal_failed_command.py:47]: Ack

**Comment by aujjwal-redhat** [comment]
```sh

Results for afr
Test nature : disruptive
TestName : test_healed_and_heal_failed_command
+-------------+-------------+-----------------------+-------------+
| Volume Type | Test Result | Time Taken (hh:mm:ss) | Skip Reason |
+-------------+-------------+-----------------------+-------------+
|     rep     |     PASS    |        00:02:20       |      NA     |
+-------------+-------------+-----------------------+-------------+
```

**Review by nik-redhat — APPROVED** [review]

**Pull request merged**